client/packages/requisitions/src/RnRForms/api/hooks/useRnRFormContext.ts

```diff
@@ -18,6 +18,7 @@ export const useRnRFormContext = create<RnRFormContext>(set => ({
     })),
   clearDraftLine: id =>
     set(state => {
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
This is a bit odd, because we have a lint rule to ignore unused errors that are "_":
```
    '@typescript-eslint/no-unused-vars': [
      'error',
      {
        argsIgnorePattern: '^_',
        caughtErrorsIgnorePattern: '^_',
      },
    ],
```
Maybe have a quick look what's going on there? But probably don't spend too much time on it, this is fine otherwise.